examples/Example5.elm

```diff
@@ -12,7 +12,12 @@ import Html exposing (Html)
 
 main : Program () Model Msg
 main =
-    sandbox { init = init, view = view, update = update }
+    element { init = init, view = view, update = update, subscriptions = subscriptions }
+
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    Dropdown.onOutsideDropdownClick model.dropdownState DropdownMsg
```

Comment:
It looks waaaay better now, thanks @jouderianjr! 😍

Reply:
maybe we can leave it at `onOutsideClick` since we are already saying `Dropdown` in the import? 🤔 🤣